presto-main/src/main/java/io/prestosql/sql/planner/plan/TableWriterNode.java

```diff
@@ -61,6 +64,7 @@ public TableWriterNode(
             @JsonProperty("fragmentSymbol") Symbol fragmentSymbol,
             @JsonProperty("columns") List<Symbol> columns,
             @JsonProperty("columnNames") List<String> columnNames,
+            @JsonProperty("notNullColumnSymbols") Set<Symbol> notNullColumnSymbols,
```

Comment:
Why use symbols here? Seems like it would be easier to have this be the column names